scripts/globals/spells/black/poisonga_ii.lua

```diff
@@ -13,16 +13,9 @@ end
 
 spellObject.onSpellCast = function(caster, target, spell)
     local dINT = caster:getStat(xi.mod.INT) - target:getStat(xi.mod.INT)
-
     local skill = caster:getSkillLevel(xi.skill.ENFEEBLING_MAGIC)
-    local power = math.max(skill / 20, 4)
-    if skill > 400 then
-        power = math.floor(skill * 49 / 183 - 55) -- No cap can be reached yet
-    end
-    power = xi.magic.calculatePotency(power, spell:getSkillType(), caster, target)
-
-    local duration = xi.magic.calculateDuration(120, spell:getSkillType(), spell:getSpellGroup(), caster, target)
-
+    local power = 7
```

Comment:
Source: I made it up.

floor(2.5 * Poisonga's damage) since Poison -> Poison II is 4 -> 10.
Use Poison II's duration since Poisonga uses Poison's duration.

Reply:
I think we should get a little more evidence than this.
https://ffxiclopedia.fandom.com/wiki/Poisonga_II?oldid=171030
Here's a list of mobs that cast it. Perhaps looking for captures fighting them and witnessing the values would be solid evidence.

Reply:
Looks like [10 damage/tick](https://youtu.be/iuA141-0-po?t=210).  Damn is confirmed poisoned from the chat log, takes 10 damage a tick. From [this video](https://www.youtube.com/watch?v=iERBSeEGSbQ) looks like it lasts 2 minutes. Poisoned at ~3:35 and poison is still on as the video fades out at 5:27.

Thanks!